client/views/directives/previewpictures.html

```diff
@@ -0,0 +1,4 @@
+<i
```

Comment:
Not sure if there is a `cursor: pointer` on this, but it should have. Also it's better to use some trigger element like `button` or `a` as wrapper.